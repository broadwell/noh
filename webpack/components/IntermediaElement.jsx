import React from "react";
import PropTypes from "prop-types";

const IntermediaElement = props => (
  <div className="intermedia__element">
    <div className="intermedia__label">{props.fieldName}</div>
    <div className="intermedia__value">{props.fieldValue}</div>
  </div>
);

IntermediaElement.propTypes = {
  fieldName: PropTypes.string.isRequired,
  fieldValue: PropTypes.string.isRequired
};

export default IntermediaElement;
