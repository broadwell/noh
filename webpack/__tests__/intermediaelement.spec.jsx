import React from "react";
import { shallow } from "enzyme";
import IntermediaElement from "../components/IntermediaElement";

describe("<IntermediaElement>", () => {
  it("renders as expected", () => {
    const component = shallow(
      <IntermediaElement fieldName="Percussion" fieldValue="Non-congruent" />
    );
    expect(component).toMatchSnapshot();
  });
});
