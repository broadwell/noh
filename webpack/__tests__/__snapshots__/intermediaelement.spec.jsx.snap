// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntermediaElement> renders as expected 1`] = `
<div
  className="intermedia__element"
>
  <div
    className="intermedia__label"
  >
    Percussion
  </div>
  <div
    className="intermedia__value"
  >
    Non-congruent
  </div>
</div>
`;
